### ISSUE TITLE
default sirepo sim_type to myapp only

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#6632]
I think by default, the feature_config.sim_types value should only be "myapp". 

And the invalid sim type message should be different if the sim_type is valid, but not enabled.

That way if someone wanted to try sirepo/elegant or similar code the output would not be this:

```
$ python
>>> import sirepo.lib
>>> sirepo.lib.Importer('elegant')
...
ModuleNotFoundError: No module named 'pymeshlab'
```

or this:

```
$ python
>>> import sirepo.lib
>>> sirepo.lib.Importer('elegant')
...
AssertionError: invalid simulation type=elegant
```

but something like this if the app was available, but not configured:

```
$ python
>>> import sirepo.lib
>>> sirepo.lib.Importer('elegant')
...
AssertionError: sim_type 'elegant' is not enabled
    You may enable this sim_type by adding 'elegant' to the SIREPO_FEATURE_CONFIG_SIM_TYPES environment variable.
    ex. export SIREPO_FEATURE_CONFIG_SIM_TYPES=elegant:opal:srw:madx:genesis
    or in a jupyter notebook:
    %set_env SIREPO_FEATURE_CONFIG_SIM_TYPES=elegant:opal:srw:madx:genesis
```

**Comment by robnagler** [comment]
I think there are two behaviors. sirepo.lib by default would typically operate on one code. sirepo.server operates on many codes. The latter is the general case, and there are too many places to maintain that alpha/prod list so we shouldn't change that behavior.

If you import sirepo.lib, it could defer all other sirepo imports until actually executing. Then sirepo.lib.Importer could call `sirepo.feature.cfg(code_being_imported)` explicitly . This would be the default configuration for the program/process.

**Comment by moellep** [comment]
Yes I agree - the sirepo.lib should load the code explicitly and not use the default configured sim_types behavior. I'll update this issue's title.
